cpp/src/io/parquet/reader_impl.cu

```diff
@@ -1032,6 +1032,41 @@ void reader::impl::decode_page_headers(hostdevice_vector<gpu::ColumnChunkDesc>&
   pages.device_to_host(stream, true);
 }
 
+__global__ void decompress_check_kernel(gpu_inflate_status_s* outputs,
+                                        size_t num_pages,
```

Comment:
We've trying to move away from passing ptr + size in favour of `device_span`

Reply:
Fixed - updated to use device_span where I have used raw pointer and size.